GDJS/Runtime/gd.js

```diff
@@ -179,7 +182,7 @@ gdjs.registerGlobalCallbacks = function() {
 
   var totalprop = 0;
 
-  innerRegisterGlobalCallbacks = function(obj, nestLevel) {
+  const innerRegisterGlobalCallbacks = function(obj, nestLevel) {
```

Comment:
Use var, not const, because of Android 4.x support: https://github.com/4ian/GDevelop/blob/master/newIDE/docs/Supported-JavaScript-features-and-coding-style.md